Platform/AlderlakeBoardPkg/Library/Stage1ABoardInitLib/Stage1ABoardInitLib.c

```diff
@@ -119,6 +121,95 @@ EarlyPlatformDataCheck (
   }
 }
 
+/**
+  CAR created by ACM covers only Initial code region.
+  This function will expand CAR to cover MRC data and improve performance of MRC code.
+
+**/
+VOID
+EFIAPI
+ExpandCacheAsRam (
```

Comment:
From the function logic, it just sets MRC data in flash to be catchable.  I didn't see it has any relation with CacheAsRam (NEM). So prefer not using it.

Reply:
Thanks for review, maybe ExpandTempRam ()? Then it can also be used if there are other features that need to set var MTRR.

Reply:
Sounds good, patch updated.